apps/mobile/src/features/approver/approver.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react';
+
+import { HasChildren } from '@/utils/types';
+import { t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import { ArrowLeftIcon, Button, Theme } from '@leather.io/ui/native';
+
+import { ApproverActions } from './components/approver-actions';
+import { ApproverAdvanced } from './components/approver-advanced';
+import { ApproverContainer } from './components/approver-container';
+import { ApproverFooter } from './components/approver-footer';
+import { ApproverHeader } from './components/approver-header';
+import { ApproverSection } from './components/approver-section';
+import { AccountCard } from './components/cards/account-card';
+import { CodeCard } from './components/cards/code-card';
+import { FeeCard } from './components/cards/fee-card';
+import { InputsAndOutputsCard } from './components/cards/inputs-outputs-card';
+import { OutcomesCard } from './components/cards/outcomes-card';
+
+export function Approver({ children }: HasChildren) {
```

Comment:
Can we colocate this in `packages/ui/src/components/approver` though, or are you just working in mobile first and will migrate later for ease of dev?

Reply:
for example, I think we can even reuse the same approver context for both implementations I think? As there's no platform specific behaviour inside (at least there shouldn't be)